dev/FHIR3-0-2-Geboortezorg/Zwangerschapskaart/_reference/Vrouw1-Kaart1/medmij-gbz-zib-AlcoholUse228273003-0d47afe5-2063-11ec-1751-020000000000.xml

```diff
@@ -1,66 +1,66 @@
-<Observation xmlns="http://hl7.org/fhir">
-  <id value="0d47afe5-2063-11ec-1751-020000000000"/>
-  <meta>
-    <profile value="http://fhir.nl/fhir/StructureDefinition/nl-core-observation"/>
-    <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AlcoholUse"/>
-  </meta>
-  <text>
-    <status value="extensions"/>
-    <div xmlns="http://www.w3.org/1999/xhtml">
-      <table>
-        <caption>Observatie/bepaling. Subject: Hanna H XXX_Kooij. Id: 7d62fc67-a6d9-11ed-8464-020000000000 (urn:oid:2.16.840.1.113883.2.4.3.11.999.7.6), Status: definitief</caption>
-        <tfoot>
-          <tr>
-            <td colspan="2">tbv conceptie al gestopt met alcohol consumptie</td>
-          </tr>
-        </tfoot>
-        <tbody>
-          <tr>
-            <th>Context</th>
-            <td>prenatale controle T-139D Vrouw1</td>
-          </tr>
-          <tr>
-            <th>Code</th>
-            <th>Waarde</th>
-          </tr>
-          <tr>
-            <td>
-              <span title="bevinding betreffende alcoholgebruik (bevinding) (228273003 - SNOMED CT)">bevinding betreffende alcoholgebruik (bevinding)</span>
-            </td>
-            <td>
-              <span title="drinkt momenteel geen alcohol (bevinding) (105542008 - SNOMED CT)">drinkt momenteel geen alcohol (bevinding)</span>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  </text>
-  <identifier>
-    <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.7.6"/>
-    <value value="7d62fc67-a6d9-11ed-8464-020000000000"/>
-  </identifier>
-  <status value="final"/>
-  <code>
-    <coding>
-      <system value="http://snomed.info/sct"/>
-      <code value="228273003"/>
-      <display value="bevinding betreffende alcoholgebruik (bevinding)"/>
-    </coding>
-  </code>
-  <subject>
-    <reference value="Patient/HANNAHXXX-KOOIJ"/>
-    <display value="Hanna H XXX_Kooij"/>
-  </subject>
-  <context>
-    <reference value="Encounter/0d47afdc-2063-11ec-1751-020000000000"/>
-    <display value="prenatale controle T-139D Vrouw1"/>
-  </context>
-  <valueCodeableConcept>
-    <coding>
-      <system value="http://snomed.info/sct"/>
-      <code value="105542008"/>
-      <display value="drinkt momenteel geen alcohol (bevinding)"/>
-    </coding>
-  </valueCodeableConcept>
-  <comment value="tbv conceptie al gestopt met alcohol consumptie"/>
-</Observation>
+<Observation xmlns="http://hl7.org/fhir">
+  <id value="0d47afe5-2063-11ec-1751-020000000000"/>
+  <meta>
+    <profile value="http://fhir.nl/fhir/StructureDefinition/nl-core-observation"/>
+    <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AlcoholUse"/>
```

Comment:
It's allowed to have two profiles, but could you explain the reason?

Reply:
For this particular Observation it might not be necessary, but for birthcare specific Observations we mandate the nl-core profile in the meta data and encourage the addition of the birthcare specific profile. Reason is that it is easier for implementers if they only have to include the generic profile instead of including specific profiles if they use the same FHIR instance for more than one use case.